torch/fx/node.py

```diff
@@ -247,6 +246,16 @@ def __init__(self, graph: 'Graph', name: str, op: str, target: 'Target',
         # transformations. This metadata is preserved across node copies
         self.meta : Dict[str, Any] = {}
 
+    def __getstate__(self):
+        return self.__dict__, self._erased, self._prev, self._next
+
+    def __setstate__(self, state):
+        d, erased, prev, next = state
+        self.__dict__.update(d)
+        self._erased = erased
+        self._prev = prev
+        self._next = next
```

Comment:
Is this BC with the old pickle format?

Reply:
Yes, it would be possible to implement the pickling for NodeBase in C++ and call that here but i didnt think it was necessary

Reply:
no, I don't care about having this in C++, just that it's BC/FC

Reply:
I don't believe this is BC with old format.  Since the old format was not a tuple of 4 items.

Reply:
Should I shove erased, prev, next into the `__dict__` and then remove them in setstate?